### PR TITLE
liburing 2.2 (new formula)

### PR DESCRIPTION
**Pull request by armanbilge**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/liburing.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+class Liburing < Formula
+  desc "This is the io_uring library, liburing"
```
Comment (armanbilge) [inline Formula/liburing.rb:2]: The official description:
```
This is the io_uring library, liburing. liburing provides helpers to setup and
teardown io_uring instances, and also a simplified interface for
applications that don't need (or want) to deal with the full kernel
side implementation.
```

File: Formula/liburing.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  homepage "https://github.com/axboe/liburing"
+  url "https://github.com/axboe/liburing/archive/refs/tags/liburing-2.2.tar.gz"
+  sha256 "e092624af6aa244ade2d52181cc07751ac5caba2f3d63e9240790db9ed130bbc"
+  license "MIT"
```
Comment (armanbilge) [inline Formula/liburing.rb:6]:
```
All software contained within this repo is dual licensed LGPL and MIT, see
COPYING and LICENSE, except for a header coming from the kernel which is
dual licensed GPL with a Linux-syscall-note exception and MIT, see
COPYING.GPL and <https://spdx.org/licenses/Linux-syscall-note.html>.
```

File: Formula/liburing.rb (hunk 1 above)
Comment (carlocab) [inline Formula/liburing.rb:2]:
```suggestion
  desc "helpers to setup and
teardown io_uring instances"
```

File: Formula/liburing.rb [hunk 3]
```diff
@@ -0,0 +1,31 @@
+class Liburing < Formula
+  desc "helpers to setup and
+teardown io_uring instances"
```
Comment (carlocab) [inline Formula/liburing.rb:3]:
```suggestion
  desc "Helpers to setup and teardown io_uring instances"
```

File: Formula/liburing.rb (hunk 2 above)
Comment (SMillerDev) [reply Formula/liburing.rb:6]: Something like this than.
```suggestion
  license any_of: ["MIT", "LGL-3.0-only"]
```

**Review by carlocab — APPROVED** [review]

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
Thanks @armanbilge!!

**Pull request closed without merging**